src/library/FOSSBilling/CentralAlerts.php

```diff
@@ -42,7 +42,7 @@ public function getDi(): ?Container
      * The alerts are **never** displayed to the clients.
      *
      * @return array The alerts
-     * @throws \Box_Exception
+     * @throws Exception
```

Comment:
Should be `@throws \FOSSBilling\Exception`, no?

Reply:
Depends on how you look at it.
Because these files are under the FOSSBilling namespace, `Exception` is actually the FOSSBilling exception class and `\Exception` would be the default PHP exception class. When moused over, my IDE correctly identified it as the FOSSBilling version as well.

If it's confusing I can change it, though

Reply:
I'm OK either way, though given the risk of confusion with `\Exception` it may be an idea to clarify for those quickly glancing at  thecode. In other cases where there is no potential confusion, then certainly agree about not needing to specify the namespace.

I mainly noticed as it is specified as `\FOSSBilling\Exception` in other reviewed files, so more of a consistency issue. So I guess more of a let's pick one way going forward and stick with it?

Reply:
Well that consistency difference is because the files outside of our library aren't namespaced under FOSSBilling.

I can either update those to add in a use statement for the `\FOSSBilling\Exception` class or adjust the library to reference the full class name to resolve that

Reply:
> Well that consistency difference is because the files outside of our library aren't namespaced under FOSSBilling.
> 
> I can either update those to add in a use statement for the `\FOSSBilling\Exception` class or adjust the library to reference the full class name to resolve that

I see no reason to change files outside of the namespace for this - consistency within the namespace is fine with me. 

For example, I can briefly see `FOSSBilling\ExtensionManager.php`, `FOSSBilling\Update.php`, `FOSSBilling\Validate.php`, and `FOSSBilling\i18n.php using both forms interchangably.

Reply:
Ah - That's just a mistake because I didn't catch all of them.
I've corrected the mentioned files now, thanks for the catch